middle_end/flambda2/terms/code_metadata.mli

```diff
@@ -33,9 +33,9 @@ module type Code_metadata_accessors_result_type = sig
 
   val params_arity : 'a t -> Flambda_arity.t
 
-  val num_leading_heap_params : 'a t -> int
+  (* val num_leading_heap_params : 'a t -> int *)
 
-  val num_trailing_local_params : 'a t -> int
+  val first_complex_local_param : 'a t -> int
```

Comment:
We should have a comment somewhere saying that this is zero-based.  Here may be the right place.

Reply:
Done.